README.md

```diff
@@ -1 +1,192 @@
-# Hello-Microverse-project
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<div align="center">
+
+  <h3><b>Set up a "Hello Microverse" Project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Set up a "Hello Microverse" project <a name="about-project"></a>
+
+
+**Set up a "Hello Microverse" project** is my first project on Microverse Program, It is focused on learning some important issues that will be used along all program experience.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li>HTML</li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li>CSS</li>
+  </ul>
+</details>
+
+<details>
+<summary>Linters</summary>
+  <ul>
+    <li>Linters</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **Creating branches on repositories.**
+- **Use of linter on HTML&CSS project.**
+- **Write a correct descriptive README file.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+
+- N/A
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+
+- N/A
+
+### Prerequisites
+
+- N/A
+
+
+### Setup
+
+- N/A
+
+
+### Install
+
+- N/A
```

Comment:
- [ ] Firstly nice making the changes here. in the reviewer's comment, he said and I quote:
 "This is a good section to have 😄 but yours is still hardcoded, most of these commands won't really help with your project so I suggest specifying the sections that don't apply. For example 👇🏽".....This means only the sections that are not needed should be left out, and he specifically mentioned Deployment.

- [ ]  There are sections there that are very important :
- From getting started to installation should be implemented. This help viewer know how to use your project 
Example 

### Setup

Clone this repository to your desired folder:
```sh
  cd my-folder
  git clone git@github.com:oovillagran/Hello-Microverse-project.git
```
### Install

```Run 
   npm install
 ```

Reply:
Hello @Shedrack-Sunday, thank you for the code review. I made corrections to the project. I hope I can continue with the next project.

Thanks in advance.